packages/@o3r/core/schematics/application/templates/package.json.template

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "<%= name %>",
+  "version": "0.0.0",
+  "scripts": {
+    "ng": "ng",
+    "start": "ng serve",
+    "build": "ng build",
+    "watch": "ng build --watch --configuration development"
+  },
+  "private": true,
+  "dependencies": {
+    "@angular/animations": "<%= rootDependencies['@angular/animations'] || rootDependencies['@angular/core'] || latest %>",
+    "@angular/common": "<%= rootDependencies['@angular/common'] || rootDependencies['@angular/core'] || latest %>",
+    "@angular/compiler": "<%= rootDependencies['@angular/compiler'] || rootDependencies['@angular/core'] || latest %>",
+    "@angular/core": "<%= rootDependencies['@angular/core'] || latest %>",
+    "@angular/forms": "<%= rootDependencies['@angular/forms'] || rootDependencies['@angular/core'] || latest %>",
+    "@angular/platform-browser": "<%= rootDependencies['@angular/platform-browser'] || rootDependencies['@angular/core'] || latest %>",
+    "@angular/platform-browser-dynamic": "<%= rootDependencies['@angular/platform-browser-dynamic'] || rootDependencies['@angular/core'] || latest %>",
+    "@angular/router": "<%= rootDependencies['@angular/router'] || rootDependencies['@angular/core'] || latest %>",
+    "rxjs": "<%= rootDependencies['rxjs'] || latest %>",
```

Comment:
Do all our o3r app depends on rxjs ?

Reply:
this is what Angular brings by default when you generate a new app https://github.com/angular/schematics-angular-builds/blob/main/workspace/files/package.json.template